server/storage/mvcc/kvstore.go

```diff
@@ -40,7 +40,7 @@ var (
 
 var restoreChunkKeys = 10000 // non-const for testing
 var defaultCompactBatchLimit = 1000
```

Comment:
nit as you suggested
```suggestion
var defaultCompactionBatchLimit = 1000
```

Reply:
👌  done